CHANGES.md

```diff
@@ -1,3 +1,61 @@
+# 0.9.2
+
+This minor release contains various bug fixes and improvements.
+
+## TODO: Bug breaking web examples
+
+## SML-style function definitions
+
+Links now supports "switch functions", a new syntax for defining functions in terms of match clauses
+directly, similar to SML. This allows writing the following function
+```
+fun ack(_,_) switch {
+  case (0, n) -> n + 1
+  case (m, 0) -> ack(m - 1, 1)
+  case (m, n) -> ack(m - 1, ack(m, n - 1))
+}
+```
+instead of the following, more verbose version:
+
+```
+fun ack(a, b) {
+  switch(a, b) {
+    case (0, n) -> n + 1
+    case (m, 0) -> ack(m - 1, 1)
+    case (m, n) -> ack(m - 1, ack(m, n - 1))
+  }
+}
+```
+
+Switch functions can also be anonymous, allowing function like the following:
+```
+fun(_, _) switch {
+  case (0, n) -> 0
+  case (m, n) -> m + n
+}
+```
+
+
+## Require OCaml 4.08
+
+The minimum required OCaml version has been raised to 4.08.
+
+
+## Miscellaneous:
+
+- The last expression in a block may no longer be followed by a semicolon (#900)
```

Comment:
I would be inclined to say more about this, since it is potentially a breaking change - perhaps making its own subsection and showing an example of how code with the (now) forbidden semicolons needs to be adapted.  Of course this is also explained in more detail in #900 .